src/core/Akka.API.Tests/CoreAPISpec.ApproveClusterTools.approved.txt

```diff
@@ -802,22 +165,22 @@ namespace Akka.Cluster.Tools.PublishSubscribe
 {
     public sealed class CurrentTopics : System.IEquatable<Akka.Cluster.Tools.PublishSubscribe.CurrentTopics>
     {
-        public readonly string[] Topics;
         public CurrentTopics(string[] topics) { }
+        public string[] Topics { get; }
```

Comment:
In this case I do not think it matters much. Wrapping it in an IReadOnlyCollection in this case only provides an illusion of safety.